Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift

```diff
@@ -1,723 +1,693 @@
-//// Automatically Generated From Tokens.swift.gyb.
-//// Do Not Edit Directly!
-//===----------------------------------------------------------------------===//
-//
-// This source file is part of the Swift.org open source project
-//
-// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
-// Licensed under Apache License v2.0 with Runtime Library Exception
-//
-// See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
-//
-//===----------------------------------------------------------------------===//
```

Comment:
Right now there is no nice way to add comments like this. 
We could add it the the `import` token